spec/presenters/two_factor_authentication/phone_selection_presenter_spec.rb

```diff
@@ -61,5 +61,15 @@
         ),
       )
     end
+
+    it 'includes a note to select an additional mfa method on first setup' do
+      expect(presenter_without_mfa.info).
+          to include(t('two_factor_authentication.two_factor_choice_options.phone_info'))
+    end
+
+    it 'does not include a note to select an additional mfa on additional setup' do
+      expect(presenter_without_mfa.info).
+          to include(t('two_factor_authentication.two_factor_choice_options.phone_info_html'))
+    end
```

Comment:
Should we be testing two different presenters here ("with MFA" and "without MFA") ? I think the `include` test is misleading, because it's likely accounting for the fact that the one string is included in the other (the "phone_info" string is a substring of "phone_info_html"). Could we use `eq` instead of `include` to make sure it's strictly the result we're expecting?

Reply:
I can use `include` instead of `eq`. My thinking testing with and without the presenter was that they are both receiving that presenter, but because it will have different information when you land on the second page, it will have information that enabled mfa counts is true as opposed to first setup